frontend/test/src/LiveMessageManager.spec.js

```diff
@@ -21,12 +21,12 @@ LiveMessageManager.prototype.setStatus = function (status) {
   setStatusOrig.call(this, status)
 
   const resolve = (this.statuscallbacks && this.statuscallbacks.shift())
-  resolve && resolve()
+  resolve && resolve(this.status)
 }
 
-LiveMessageManager.prototype.onstatuschange = function (callback) {
+LiveMessageManager.prototype.onStatusChange = async function () {
```

Comment:
I'm confused because you only `await` for this function in unit tests, I'm not sure to understand

Reply:
It is only present in unit tests.

Reply:
oh, the file is in tests folder ok.